wireapp/wire-android
# Edit Phone: Pick Country Code dialog

## What's new in this PR?

JIRA ticket: https://wearezeta.atlassian.net/browse/AN-6644

### Issues

Implemented a country code picker to change your country code 

### Testing
1. Register a user with email
2. Go to settings 
3. Add a phone number 
4. Click dialog again to edit phone number
5. Click on country button
6. Pick new country/country code 
7. Should update screen 

#### APK
[Download build #1211](http://10.10.124.11:8080/job/Pull%20Request%20Builder/1211/artifact/build/artifact/wire-dev-PR2598-1211.apk)
[Download build #1213](http://10.10.124.11:8080/job/Pull%20Request%20Builder/1213/artifact/build/artifact/wire-dev-PR2598-1213.apk)
[Download build #1215](http://10.10.124.11:8080/job/Pull%20Request%20Builder/1215/artifact/build/artifact/wire-dev-PR2598-1215.apk)
[Download build #1227](http://10.10.124.11:8080/job/Pull%20Request%20Builder/1227/artifact/build/artifact/wire-dev-PR2598-1227.apk)
[Download build #1244](http://10.10.124.11:8080/job/Pull%20Request%20Builder/1244/artifact/build/artifact/wire-dev-PR2598-1244.apk)
